hugr-py/src/hugr/dfg.py

```diff
@@ -240,14 +531,29 @@ def _wire_up_port(self, node: Node, offset: int, p: Wire) -> Type:
 
 
 class Dfg(_DfBase[ops.DFG]):
+    """Builder for a simple nested Dataflow graph, with root node of type
+    :class:`DFG <hugr.ops.DFG>`.
+
+    Args:
+        input_types: The input types of the the dataflow graph. Output types are
+        calculated by propagating types through the graph.
+        extension_delta: The extension delta of the graph.
```

Comment:
Maybe needs more explanation or a reference to an explanation of what this means?

Reply:
i think I'm just going to remove until we have a better story about extensions - the rest of the code/tests etc. just ignores them